python/graphstorm/model/node_glem.py

```diff
@@ -64,6 +71,34 @@ def __init__(self,
         # True: lm is being trained
         # False: gnn is being trained
         self.training_lm = None
+        if lm_param_group is None:
+            lm_param_group = []
+        if gnn_param_group is None:
+            gnn_param_group = []
```

Comment:
what is the default behavior?

Reply:
It's determined in `argument.py`. If empty list, it will be `default_parameter_groups`: 

```
{
    'lm': list(self.lm.decoder.parameters()),
    'gnn': list(self.gnn.gnn_encoder.parameters())
}
```